Cuis-University-COOP.pck.st

```diff
@@ -1458,6 +1572,51 @@ withSelector: messageName applyCondition: aMessageNode
 	
 	or: [ aMessageNode isMessage: messageName receiver:  self literalZero arguments: self messageSize] .! !
 
+!COOPRuleInstanceMethodsToClass methodsFor: 'testing' stamp: 'GET 11/7/2019 23:54:45'!
+applyCondition: aMessageNode 
+	
+	| aClass metaClass class |
+	aClass _ self canAnalize:aMessageNode ifNone: [ ^ false ].
+	metaClass _ aClass theMetaClass .
+	class _ aClass theNonMetaClass .
+	
+	^ (self selector: aMessageNode selectorSymbol IsInClass: class ) and: [ ( self selector: aMessageNode selectorSymbol IsInClass: metaClass ) not ] ! !
+
+!COOPRuleInstanceMethodsToClass methodsFor: 'testing' stamp: 'GET 11/8/2019 00:06:45'!
+canAnalize: aMessageNode ifNone: aBlockClosure 
+	 | receiver |
+	receiver _  aMessageNode receiver.
+	
+	^ (receiver notNil 
```

Comment:
Podes identar poniendo el #and en la misma linea y el #ifFalse:ifTrue: en la de abajo. Te queda así en dos lineas re cheto